src/widgets/DataTable/DataTableRow.js

```diff
@@ -120,6 +120,7 @@ const DataTableRow = React.memo(
                   cellTextStyle={editableCellText}
                   rowIndex={rowIndex}
                   placeholder={placeholder}
+                  focusedColumn={SUSSOL_ORANGE}
```

Comment:
hmm the `TextInputCell` doesn't have a prop `focusedColumn` - I think this should be `onFocus` prop which you can pass a function which dispatches the `focusCell` action, is that what you're goin for?